app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if type(other) == int or type(other) == float:
+            return Distance(
+                self.km + other
+            )
+        return Distance(
+            self.km + other.km
+        )
+
+    def __iadd__(self, other):
+        if type(other) == int or type(other) == float:
+            self.km += other
+            return self
+        self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(
+            round(self.km / other, 2)
+        )
+
+    def __lt__(self, other):
+        if type(other) == int or type(other) == float:
+            return self.km < other
+        return self.km < other.km
+
+    def __gt__(self, other):
+        if type(other) == int or type(other) == float:
+            return self.km > other
+        return self.km > other.km
+
+    def __eq__(self, other):
+        if type(other) == int or type(other) == float:
+            return self.km == other
+        return self.km == other.km
+
+    def __le__(self, other):
```

Comment:
You can also optimize `__le__` and `__ge__` methods by returning `not __gt__` and `not __lt__` respectively.
It was on today's lecture, actually. :)